src/main/java/com/example/demo/service/LoginService.java

```diff
@@ -0,0 +1,31 @@
+package com.example.demo.service;
+
+import com.example.demo.controller.dto.request.LoginRequest;
+import com.example.demo.controller.dto.response.MemberResponse;
+import com.example.demo.domain.Member;
+import com.example.demo.exception.ApplicationException;
+import com.example.demo.repository.MemberRepository;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Service;
+
+import static com.example.demo.exception.ErrorCode.MEMBER_NOT_FOUND;
+
+@Service
+public class LoginService {
+
+    private final MemberRepository memberRepository;
+    private final PasswordEncoder passwordEncoder;
+
+    public LoginService(MemberRepository memberRepository, PasswordEncoder passwordEncoder) {
+        this.memberRepository = memberRepository;
+        this.passwordEncoder = passwordEncoder;
+    }
```

Comment:
```suggestion
@Service
@RequiredArgsConstructor
public class LoginService {

    private final MemberRepository memberRepository;
    private final PasswordEncoder passwordEncoder;
```
소소한 팁인데, `@RequiredArgsConstructor`를 사용하면 final( + NotNull) 선언한 변수들에 대해서 생성자 주입을 해준답니다.
코드 짤 때 더 간결해지고, 좋아요!
[블로그 참고](https://velog.io/@developerjun0615/Spring-RequiredArgsConstructor-%EC%96%B4%EB%85%B8%ED%85%8C%EC%9D%B4%EC%85%98%EC%9D%84-%EC%82%AC%EC%9A%A9%ED%95%9C-%EC%83%9D%EC%84%B1%EC%9E%90-%EC%A3%BC%EC%9E%85)